src/components/atoms/UiLink/UiLink.stories.js

```diff
@@ -144,3 +144,34 @@ LinkSecondary.args = { modifiers: [ 'ui-button--theme-secondary' ] };
 export const LinkOnBrand = Template.bind({});
 LinkOnBrand.args = { modifiers: [ 'ui-button--theme-brand' ] };
 LinkOnBrand.parameters = { backgrounds: { default: 'brand' } };
+
+export const LinkInTheLongText = (args) => ({
```

Comment:
🛑 @ArkadiuszKrzymienski here we have a blocker. We can't use `flex` or `inline-flex` for UiLink. UiLink is different than UiButton (event if is ui-button--text). UiLink is a part of <UiText /> (I found this error because we have link in SidePanel stories).

We should keep old approach in UiLink and maybe icon should be `--end` instead `--right`